composites/Plugin/SnippetEditor/serialization.js

```diff
@@ -64,128 +66,240 @@ export function serializeEditor( rawContent ) {
 }
 
 /**
- * Unserializes a tag into a string.
+ * Determines the title for a given name.
```

Comment:
To understand what this actually means I'm forced to look at the code. I'd suggest to clarify a bit.